servant-client-core/servant-client-core.cabal

```diff
@@ -1,27 +1,30 @@
+cabal-version:       >=1.10
 name:                servant-client-core
 version:             0.15
+
 synopsis:            Core functionality and class for client function generation for servant APIs
+category:            Servant, Web
```

Comment:
Oh we get our own category now? =)

Reply:
we had it already, since 0.13 or 0.14: http://hackage.haskell.org/packages/#cat:Servant